src/components/Loading/index.jsx

```diff
@@ -1,6 +1,43 @@
 import React from 'react'
-import LoadingStyled from './styled'
+import { makeStyles } from '@material-ui/core/styles'
+import CircularProgress from '@material-ui/core/CircularProgress'
 
-const Loading = () => <LoadingStyled>Loading...</LoadingStyled>
+const useStyles = makeStyles(theme => ({
+  progress: {
+    margin: theme.spacing(2)
+  }
+}))
+
+// positions loader in the center of the containing div element
+// enlarges animation for better visibility
+const loaderStyle = {
```

Comment:
Styles should be written in 1 file named style.js as per our standard:

**style.js**
```
const LoadingStyled = styled.div`
  width: 100%;
  height: 100vh;
  display: flex;
  align-items: center;
  justify-content: center;
`

export default LoadingStyled
```